src/bin/add/args.rs

```diff
@@ -106,6 +109,22 @@ impl Args {
 
         Ok(vec![dependency])
     }
+
+    fn get_upgrade_prefix(&self) -> &'static str {
+        match self.flag_upgrade {
+            None => &"",
+            Some(ref u) => match u.to_uppercase().as_ref() {
+                "NONE" => &"=",
+                "PATCH" => &"~",
+                "MINOR" => &"^",
+                "ALL" => &">=",
```

Comment:
Why did you prefix all the strings with `&`? A static string slice is already of type `&'static str`, so there is no need to borrow it twice :)